### PR TITLE
minsung 주택가격예측

### PR DESCRIPTION
**Pull request by mildstar**

### CODE REVIEW

File: mildstar/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Pipfile
+Pipfile.lock
+MLstudy
```
Comment (HyunggyuJang) [inline mildstar/.gitignore:3]: What is MStudy?

**Review by HyunggyuJang — COMMENTED** [review]
Just gets curious

File: mildstar/.gitignore (hunk 1 above)
Comment (mildstar) [reply mildstar/.gitignore:3]: Idk

**Review by HyunggyuJang — APPROVED** [review]
LGTM!

**Comment by HyunggyuJang** [comment]
Closes as it was test pull request, and isn't via valid pull request process (source branch should be from forked repository, not the **this** repository).

**Pull request closed without merging**